slide-deck.js

```diff
@@ -321,7 +367,7 @@ class slideDeck extends HTMLElement {
     ...this.shadowRoot.querySelectorAll(`button[${attr}]`),
   ];
 
-  #getButtonValue = (btn, attr) => btn.getAttribute(attr) || btn.innerText;
+  #getButtonValue = (btn, attr) => btn.getAttribute(attr) || btn.innerText.trim();
```

Comment:
Do we want to also add a `trim` to the attribute? Potentially add a `clean` method that trims and lowercases the input?